permissions/plugin-audit-log.yml

```diff
@@ -0,0 +1,10 @@
+---
+name: "audit-log"
+github: "jenkinsci/audit-log-plugin"
+paths:
+  - "io/jenkins-ci/plugins/audit-log"
```

Comment:
didn't you mean to remove the ci suffix? What I talked about was actually `io.jenkins.plugins`, also because I explained we were getting away from hinting that Jenkins is "CI" only.

We don't enforce that, note. So if that is really what you meant, we can move forward.

cc @jvz

Reply:
@batmat thanks for catching that. i just made the corrections- both here and in our plugin.